RFCs/FS-1128-allow-static-members-in-interfaces.md

```diff
@@ -0,0 +1,49 @@
+# F# RFC FS-1128 - Allow static members in interfaces
+
+The design suggestion [Allow static members in interfaces](https://github.com/fsharp/fslang-suggestions/issues/1191) has been marked "approved in principle".
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/1191)
+- [x] Approved in principle
+- [ ] Implementation
+- [ ] Design Review Meeting(s) with @dsyme and others invitees ???
+- [ ] Discussion ???
```

Comment:
Typically, you'd create a discussion thread (click "Discussion") and add the link here.